definitions/tools/vep.cwl

```diff
@@ -28,16 +26,11 @@ inputs:
             prefix: "-i"
             position: 1
     cache_dir:
-        type: string?
+        type: string
         inputBinding:
             valueFrom: |
                 ${
-                    if (inputs.cache_dir) {
-                        return ["--offline", "--cache", "--dir", inputs.cache_dir ]
-                    }
-                    else {
-                        return "--database"
-                    }
+                    return ["--offline", "--cache", "--dir", inputs.cache_dir ]
```

Comment:
Now that this is required, we could move `--offline` and `--cache` to the arguments and make this a normal `prefix: "--dir"` parameter without the `valueFrom`.

Reply:
Updated in the latest commit fdad71a